common/services/person.test.js

```diff
@@ -21,12 +22,12 @@ const mockPerson = {
 }
 
 describe('Person Service', function () {
-  describe('#format()', function () {
+  describe('.format()', function () {
     context('when relationship field is string', function () {
       let formatted
 
       beforeEach(async function () {
-        formatted = await personService.format({
+        formatted = await PersonService.format({
```

Comment:
This is another result of the static in the service - the methods should be on the instance, not on the class, so here you should be using the `personService` instance rather than the class - the class doesn't have access to the api client in the req.